queue_services/entity-emailer/tests/unit/email_processors/test_filing_notification.py

```diff
@@ -101,12 +119,12 @@ def test_maintenance_notification(app, session, status, filing_type):
     token = 'token'
     # test processor
     with patch.object(filing_notification, '_get_pdfs', return_value=[]) as mock_get_pdfs:
-        with patch.object(filing_notification, 'get_recipients', return_value='test@test.com') \
+        with patch.object(filing_notification, 'get_recipients', return_value=testing_email) \
           as mock_get_recipients:
             email = filing_notification.process(
                 {'filingId': filing.id, 'type': filing_type, 'option': status}, token)
 
-            assert 'test@test.com' in email['recipients']
+            assert testing_email in email['recipients']
             assert email['content']['body']
             assert email['content']['attachments'] == []
```

Comment:
I only changed it here because before I changed anything the test was failing. I checked the conditions and it seemed to be a wrong asserting for 'comp_party@email.com'.